simulationfactory/src/util/Stylesheet.js

```diff
@@ -1,55 +1,52 @@
-import { makeStyles, useTheme } from "@material-ui/core/styles";
-
+import { createMuiTheme, makeStyles } from "@material-ui/core/styles";
+import palette from "./palette.json"
+export const Theme = createMuiTheme(palette);
 const drawerWidth = 240;
 
 function CreateStyles() {
-  const theme = useTheme();
-  let styles = makeStyles((theme) => ({
+  const styles = makeStyles((Theme) => ({
     root: {
       flexGrow: 1,
       display: "flex",
     },
+    title: {
+      color: Theme.palette.primary.main,
+    },
     card: {
-      padding: theme.spacing(2),
+      padding: Theme.spacing(2),
       textAlign: "center",
-      color: theme.palette.text.secondary,
     },
     drawer: {
-      [theme.breakpoints.up("sm")]: {
+      [Theme.breakpoints.up("sm")]: {
         width: drawerWidth,
         flexShrink: 0,
       },
     },
     appBar: {
-      [theme.breakpoints.up("sm")]: {
+      [Theme.breakpoints.up("sm")]: {
         width: `calc(100% - ${drawerWidth}px)`,
         marginLeft: drawerWidth,
       },
     },
     menuButton: {
-      marginRight: theme.spacing(2),
-      [theme.breakpoints.up("sm")]: {
+      marginRight: Theme.spacing(2),
+      [Theme.breakpoints.up("sm")]: {
         display: "none",
       },
     },
     // necessary for content to be below app bar
-    toolbar: theme.mixins.toolbar,
+    toolbar: Theme.mixins.toolbar,
     drawerPaper: {
       width: drawerWidth,
     },
     content: {
       flexGrow: 1,
-      padding: theme.spacing(3),
-    },
-    blue: {
-      color: "blue",
+      padding: Theme.spacing(3),
     },
     defaultButton: {
       variant: "contained",
-      color: "primary",
       size: "medium"
     },
```

Comment:
This class has no effect on the appearance of the buttons. We should probably remove it, and stop referencing it on all the buttons.
I'm like 90% sure I made this class, and forgot to delete it. That's my bad.

Reply:
Good to go, I will take a look at it either later this evening or tomorrow morning!